cmd/list/installed/installed.go

```diff
@@ -34,7 +38,11 @@ func List() error {
 			return fmt.Errorf("failed to determine if '%s' has been installed: %w", t.Name(), err)
 		}
 		if installed {
-			fmt.Printf("- %s\n", t.Name())
+			installedVersion, err := t.InstalledVersion(installDir, latestDir)
+			if err != nil {
+				return fmt.Errorf("failed to determine version for '%s': %w", t.Name(), err)
```

Comment:
Would it make sense to log the error, but keep looping (don't `return`)?

Reply:
Not sure how we could handle that if we ignore this, and I do not see why those functions would fail. If it happen, I guess we would have issue everywhere...

Reply:
Looking deeper at it, LatestDir() will fail if InstallDir() fails because os.UserHomeDir() fails!
I guess we can agree that is not really an option to consider, even if we have to handle the err returned...